aleemer/Internify
# [IN-51] Structure three columns in Create page

#### User Story
[[IN-51]](https://chaosneutral.atlassian.net/jira/software/projects/IN/boards/1?selectedIssue=IN-51)

#### Changes made
- Added 3 Grid items in the `Create` page so that the page has consistent spacing
- Updated the 'Back' button
- Added margin to the bottom of the page to balance negative space
- Fixed warning messages from dispatching actions to GET students

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/37882255/125008281-09e7fc00-e017-11eb-944f-660844f7aa84.png">

#### Does your new code introduce new warnings on dev console?
No

#### Test Steps
1. Navigate to the `Create` page and navigate through the steps, you should see a consistency in the spacing and structure of all the columns
2. On the first step the Back button should not appear, and it should reappear when you hit continue
